drivers/sensor/adltc2990/adltc2990.h

```diff
@@ -18,14 +19,14 @@ enum adltc2990_monitor_pins {
 	V4,
 	INTERNAL_TEMPERATURE,
 	SUPPLY_VOLTAGE
-} adltc2990_monitor_pins;
+};
 
 enum adltc2990_monitoring_type {
 	NOTHING,
 	VOLTAGE_DIFFERENTIAL,
 	VOLTAGE_SINGLEENDED,
 	TEMPERATURE
-} adltc2990_monitoring_type;
+};
```

Comment:
Looks like this change should move from the bug fix commit to the refactor commit

Reply:
done